source

```diff
@@ -103571,8 +103572,9 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
     state</span>.
     <!-- "=" added because of https://www.w3.org/Bugs/Public/show_bug.cgi?id=9207#c5 -->
 
-    <p>If the last character matched is not a U+003B SEMICOLON character (;), this is a <span>parse
-    error</span>.</p>
+    <p>If the last character matched is not a U+003B SEMICOLON character (;), this is an
+    <dfn data-x="parse-error-unexpected-character-after-reference-name">unexpected-character-after-reference-name</dfn>
```

Comment:
This one is explained in example after this state: the issue is if you miss `;` after character reference this can lead to ambiguity. Therefore, it's `missing-semicolon-after-character-reference` (it's also will be reused later for numeric character references)